<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Summary</class>
 <widget class="QWidget" name="Summary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Verb Paradigm Mode</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>800</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget {
	background-color: #83A2FF;
	color: #FFFFFF;
}</string>
   </property>
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>701</width>
      <height>161</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 50%; /* Makes the label circular */
border: 2px solid #000000; /* Border color and width */
padding: 10px; /* Adjust padding as needed */
background-color: #83A2FF;
color: #FFD28F;
border: none;
</string>
    </property>
    <property name="text">
     <string>Complete the Verb Paradigm to win the Game</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="check_button">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>350</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 20px; /* Adjust the radius as needed */
	background-color: #FFE3BB;
}
QPushButton:hover {
	background-color: #FFD28F;
	}
QPushButton:pressed {
	background-color: #B4BDFF;
}</string>
    </property>
    <property name="text">
     <string>Check!</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="paradigm_table">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>901</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget {
     font-size: 24px;  /* Set font size for cells */
     color: white;     /* Set text color for cells */
	border: 2px solid #000000; /* Border color and width */
	padding: 10px; /* Adjust padding as needed */
	background-color: #FFE3BB;
	color: black;
	font-weight: bold;
	border: none;
}

QHeaderView::section {
     color: white;     /* Set text color for header */
     background-color: #FFD28F; /* Set background color for header */
     border: 1px solid #555; /* Set border for header */
	height: 36px;     /* Set height for each header */
	width: 260px;
	font-weight: bold;
}</string>
    </property>
    <property name="rowCount">
     <number>1</number>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>220</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>24</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>Word to Guess</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Present</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Pr√§teritum</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Perfekt</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Italian</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="comparison_result_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>470</y>
      <width>861</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 50%; /* Makes the label circular */
border: 2px solid #000000; /* Border color and width */
padding: 10px; /* Adjust padding as needed */
background-color: #83A2FF;
color: #FFD28F;
border: none;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="go_to_summary_button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>350</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 20px; /* Adjust the radius as needed */
	background-color: #FFE3BB;
}
QPushButton:hover {
	background-color: #FFD28F;
	}
QPushButton:pressed {
	background-color: #B4BDFF;
}</string>
    </property>
    <property name="text">
     <string>Go to Summary!</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
